drivers/include/periph/flashpage.h

```diff
@@ -106,6 +122,26 @@ static inline int flashpage_page(void *addr)
  */
 void flashpage_write(int page, void *data);
 
+/**
+ * @brief   Write the given address with the given data and length
+ *
+ * @warning Make sure the targeted memory area is erased before calling
+ *          this function
+ *
+ * Both target address and data address must be aligned to
+ * @p FLASHPAGE_RAW_ALIGN. len must be a multiple of @p FLASHPAGE_RAW_BLOCKSIZE.
```

Comment:
@miri64 correct me if I'm wrong, but if I am not mistaken, `@p` is to be used for function parameters (so `@p len` here), and @ref for things like defines (as in `@ref FLASHPAGE_RAW_ALIGN`).

Reply:
True, I just checked and I need to use `@def` (or `\def`). Will change.

Reply:
No... @def for defining a `define` in doxygen. So you could do something like


```C
/**
 * @def   BLA
 * @brief blabla
 *
 * @def   FOO
 * @brief foobare
 */
#define BLA
#define FOO
```

Use `@ref` instead, as @haukepetersen pointed out.

Reply:
(this way you also create a nice link to that define in the parsed documentation ;-).

Reply:
hmmm I don't get it. The defines are already above without the `@def` doxygen command, but I can add it. The question here is, how can I reference them to create the nice link you're talking about?